wagtail/admin/views/pages/usage.py

```diff
@@ -4,38 +4,34 @@
 from django.core.exceptions import PermissionDenied
 from django.http import Http404
 from django.urls import reverse
+from django.utils.functional import cached_property, classproperty
+from django.utils.text import capfirst
+from django.utils.translation import gettext
 from django.utils.translation import gettext_lazy as _
 
-from wagtail.admin.ui.tables import Column, DateColumn
-from wagtail.admin.ui.tables.pages import (
-    PageStatusColumn,
-    PageTable,
-    PageTitleColumn,
-    ParentPageColumn,
-)
 from wagtail.admin.views import generic
 from wagtail.admin.views.generic.base import BaseListingView
+from wagtail.admin.views.generic.permissions import PermissionCheckedMixin
+from wagtail.admin.views.pages.listing import PageFilterSet, PageListingMixin
 from wagtail.admin.views.pages.utils import (
     GenericPageBreadcrumbsMixin,
 )
 from wagtail.models import Page
 
 
-class ContentTypeUseView(BaseListingView):
-    results_template_name = "wagtailadmin/pages/usage_results.html"
+class ContentTypeUseView(PageListingMixin, PermissionCheckedMixin, BaseListingView):
+    index_url_name = "wagtailadmin_pages:type_use"
+    index_results_url_name = "wagtailadmin_pages:type_use_results"
     page_title = _("Pages using")
     header_icon = "doc-empty-inverse"
-    page_kwarg = "p"
     paginate_by = 50
-    columns = [
-        PageTitleColumn("title", classname="title", label=_("Title")),
-        ParentPageColumn("parent", label=_("Parent")),
-        DateColumn("latest_revision_created_at", label=_("Updated"), width="12%"),
-        Column("type", label=_("Type"), accessor="page_type_display_name", width="12%"),
-        PageStatusColumn("status", label=_("Status"), width="12%"),
-    ]
-    table_class = PageTable
-    table_classname = "listing"
+    filterset_class = PageFilterSet
+
+    @classproperty
+    def columns(cls):
+        columns = PageListingMixin.columns.copy()
+        columns.pop(4)  # Remove the "Type" column
```

Comment:
I wonder if `BaseListingView` should allow dicts as well, so we can refer to the columns by name 🤔

Reply:
Thanks, done in 1ebf3b4.